### ISSUE TITLE
Add version info in include/c.h

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#95]
```
What steps will reproduce the problem?
1. There isn't any version info in b binding file include/c.h

What is the expected output? What do you see instead?
I can get version info in c interface

What version of the product are you using? On what operating system?
ANY


I'm working on a php binding using c binding, but I can't get version info, 
If it was supplied as a macro will be much appreciated. 
```

---

Original issue reported on code.google.com by `reeze....@gmail.com` on 13 Jun 2012 at 3:42

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.0.

```

---

Original comment by `san...@google.com` on 16 Oct 2012 at 11:30

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `san...@google.com` on 16 Oct 2012 at 11:30
- Changed state: **Fixed**
